### PR TITLE
add new recipes (reports package)

### PR DESCRIPTION
**Pull request by cokelaer**

### CODE REVIEW

File: recipes/reports/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+test:
+  imports:
+    - reports
+    - reports.resources
+
+about:
+  home: 'http://github.com/cokelaer/reports/'
+  license: LGPL-3.0
+  license: LGPL v3+
```
Comment (synapticarbors) [inline recipes/reports/meta.yaml]: The github repo for this project says that it is BSD 3-clause and the `LICENSE` file looks to match this. I just wanted to double check if LGPL was actually correct here?

Comment (cokelaer) [inline recipes/reports/meta.yaml]: You right, this is BSD. Should be fixed now.

File: recipes/reports/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - jinja2
+
+test:
+  imports:
+    - reports
+    - reports.resources
+
+about:
+  home: 'http://github.com/cokelaer/reports/'
+  license: BSD-3.0
```
Comment (synapticarbors) [inline recipes/reports/meta.yaml]: I have not seen this referred to as `BSD-3.0` before. Typically it is `BSD 3-clause`.

Comment (cokelaer) [inline recipes/reports/meta.yaml]: fixed the license name

**Comment by cokelaer** [comment]
I was wondering what is the delay for the reports package to appear on the feedstocks page ?
This is not to put any pressure but to understand the mechanism. I had the feeling that for other packages ported recently (e.g. colormap) appear quicker on conda-forge repo ? Thank you for your work by the way.

File: recipes/reports/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+{% set version = "0.3.1" %}
+{% set sha256 = "72b5d5d1f8a452f4c8b1f69862c2d6bc4ac953490273b23b232dc35692c01755" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: reports-0.3.1.tar.gz
```
Comment (jakirkham) [inline recipes/reports/meta.yaml]: Please template this out using `version`.

File: recipes/reports/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  url: https://pypi.python.org/packages/24/c0/d8d329f3a5027c3382140e0db925b789eb80615044ce4088d96da529e318/{{ name }}-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/reports/meta.yaml]: Please replace with the URL from the [`example`]( https://github.com/conda-forge/staged-recipes/blob/a93bf8b192514020243fecfa78948f1f3ccf750d/recipes/example/meta.yaml#L18 ).

**Comment by jakirkham** [comment]
> I was wondering what is the delay for the reports package to appear on the feedstocks page ?

Because it hasn't been merged yet. Possibly also people haven't had time or don't know to circle back and check on new changes. Please feel free to ping `@conda-forge/staged-recipes` if you need someone to review changes that you have made. Though please do use this judiciously.

File: recipes/reports/meta.yaml (hunk 4 above)
Comment (cokelaer) [inline recipes/reports/meta.yaml]: I triggered the CI build by incorporating your suggestions. thanks

**Comment by cokelaer** [comment]
@jakirkham @synapticarbors 

The AppVeyor build failed for one of the case (py27). I cannot see any specific errors. I noticed that sometines in the Appveyor, rebuilding may solve the issue. Is there a way to trigger the re-build of AppVeyor only (without a new commit/push) ? thanks

**Comment by cokelaer** [comment]
@conda-forge/staged-recipes the CI build does not seem to start. Not sure what to do here. I did a dummy commit but got stuck again. Please let me know what should be done. thanks

**Pull request merged**